force-app/main/default/customindex/Journey__c.Candidate_Email_Address__c.indx-meta.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> You've stumbled across a bug here - custom index metadata shouldn't be exposed, but is because of a bug in packaging. Please delete the customindex folder from your repository, or it might cause your problems when they fix it.

Make sure to delete the customindex folder.